src/main/java/com/github/maxopoly/WurstCivTools/listener/ToolListener.java

```diff
@@ -26,15 +28,15 @@ public ToolListener() {
 		this.manager = WurstCivTools.getManager();
 	}
 
-	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+	@EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
 	public void blockBreak(BlockBreakEvent e) {
 		ItemStack is = e.getPlayer().getItemInHand();
 		for (Tag tag : getTags(is)) {
 			tag.getEffect().handleBreak(e.getPlayer(), e);
 		}
 	}
 
-	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+	@EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
```

Comment:
Just asking because I think originally I had a reason behind them, but I'm not what it was or if there even was one.